devicetypes/iblinds/iblinds-zwave.src/iblinds-zwave.groovy

```diff
@@ -0,0 +1,299 @@
+/**
+ *  Copyright 2017 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+import groovy.json.JsonOutput
+
+
+metadata {
+    definition (name: "iblinds Z-Wave", namespace: "iblinds", author: "HABHomeIntel", ocfDeviceType: "oic.d.blind",  mnmn: "SmartThings", vid: "generic-shade-3") {
+
+        capability "Switch Level" 
+        capability "Switch" 
+        capability "Battery" 
+        capability "Refresh" 
+        capability "Actuator"
+        capability "Health Check" 
+        capability "Window Shade" 
+        capability "Window Shade Preset"	
+
+        command "stop"
+
+
+        fingerprint mfr:"0287", prod:"0003", model:"000D", deviceJoinName: "iBlinds Motor"
+        fingerprint mfr:"0287", prod:"0004", model:"0071", deviceJoinName: "iBlinds Motor"
+    }
+
+    simulator {
+        status "open":  "command: 2603, payload: FF"
+        status "closed": "command: 2603, payload: 00"
+        status "10%": "command: 2603, payload: 0A"
+        status "66%": "command: 2603, payload: 42"
+        status "99%": "command: 2603, payload: 63"
+        status "battery 100%": "command: 8003, payload: 64"
+        status "battery low": "command: 8003, payload: FF"
+
+        // reply messages
+        reply "2001FF,delay 1000,2602": "command: 2603, payload: 10 FF FE"
+        reply "200100,delay 1000,2602": "command: 2603, payload: 60 00 FE"
+        reply "200142,delay 1000,2602": "command: 2603, payload: 10 42 FE"
+        reply "200163,delay 1000,2602": "command: 2603, payload: 10 63 FE"
+    }
+
+    tiles(scale: 2) {
```

Comment:
Whole "tiles" section will be handled in the "SmartThings Classic" only.

Reply:
Correct,  This is from the  published Z-Wave-Window-Shade DTH